tests/testqualification/qualification/qual_workflows/2_metrics/kri0001b_custom.yaml

```diff
@@ -0,0 +1,62 @@
+meta:
+  Type: Metric
+  ID: kri0001b_custom
+  File: kri0001b_custom.yaml
+  GroupLevel: Race
```

Comment:
`gsm::analyticsInput` doesn't have GroupLevel == "Race", so i'm a bit confused how this is passing (it is- i just ran it), especially with the `spec` below.

Reply:
GroupLevel isn't used in this workflow, so thats why it works fine. So, you can disregard this comment.